style.css

```diff
@@ -171,88 +177,124 @@ video {
 
 .contact {
 	color: white;
-	background-color:#270483;
+	background-color: #270483;
 	padding-bottom: 60px;
 }
 
-.contact p {
-	font-family: 'Source Sans Pro', sans-serif;
-}
 footer {
 	color: white;
 	background-color: #270483;
-	font-family: 'Source Sans Pro', sans-serif;
 }
+
 footer p {
-	margin:0;
+	margin: 0;
+	font-size: 1rem;
 	
 }
 
 .projectBlocks {
+	display: -webkit-box;
+	display: -ms-flexbox;
 	display: flex;
+	-webkit-box-pack: space-evenly;
+	-ms-flex-pack: space-evenly;
 	justify-content: space-evenly;
+	-ms-flex-wrap: wrap;
 	flex-wrap: wrap;
 }
 
 .block {
 	margin: 20px;
-	width: 350px;
-	height: 350px;
+	width: 25vw;
+	height: 25vw;
 	background-color: #F3F1F9;
 	border: 10px solid white ;
-	/*box-shadow: 0 0px 80px rgba(86,11,233,0.4);*/
-	font-family: 'Poppins', sans-serif;
 	text-align: center;
 }
 
-
+.block h3 {
+	margin: 0 2vh 0 2vh;
+	padding: 0;
+	
+}
 
 .block img {
-	max-width:100%;
-	max-height:100%;
+	max-width: 100%;
+	max-height: 100%;
+	
 }
 
 
 
 
 
 @media only screen and (max-width: 1000px) {
-
-	.intro {
-		padding: 2%;
-	}
+	
 	.navigationBar {
 		display: grid;
 		grid-template-rows: 50% 50%;
 		grid-template-columns: 100%;
-		height: 120px;
 	}
+	
+	.navigationBar img {
+		
+	}
+	
 	.welcome {
-		padding-top: 120px;
+		padding-top: 12vh;
+		
+	}
+	.container {
+		padding-top: 12vh;
 	}
+
 	.video {
 		background-image: src("");
 		object-fit: cover;
 		width: 100vw;
 	}
 	
 	.intro {
-		padding-top: 70px;
+
 	}
+	
 	.intro h1 {
-		font-size: 3em;
-		padding:0;
+		padding: 0;
+		margin: 5%;
+		font-size: 3rem;
 	}
+	
 	.intro p {
-		font-size: 2em;
+		padding-top: 1%;
+		margin: 5%;
+		font-size: 1.5rem;
+		width: 70%;
 	}
+	
 	.bigBlock {
 		flex-direction: column;
 		padding: 0;
 	}
+	
 	.bigBlock img{
 		max-width: 100%;
 		max-height: 100%;
 	}
+
+	.block {
+		width: 80%;
+		height: 80%;
+	}
+	
+	.button {
```

Comment:
You should probably give this class a more specific name like `socialButton` because later on you might add a different type of button to your page (most buttons tend to be text buttons) and you will find that your class name for a button is taken